DuckDuckGo/MainViewController.swift

```diff
@@ -17,10 +17,11 @@ class MainViewController: UIViewController {
     @IBOutlet weak var shareButton: UIBarButtonItem!
     @IBOutlet weak var backButton: UIBarButtonItem!
     @IBOutlet weak var forwardButton: UIBarButtonItem!
-    weak var omniBar: OmniBar?
+    weak var omniBar: OmniBar!
```

Comment:
The `!` force unwraps an optional. The omniBar has to be declared as an optional/nullable variable because it isn't instantiated during construction. However it is instantiated during `viewDidLoad` and as it is now never touched again, it makes sense to force unwrap here rather than unwrap manually everywhere else. Force unwrapping is a bit of an anti-pattern but common in this case - you'll see it a lot for ViewControllers where items are instantiated by InterfaceBuilder or in the viewDidLoad phase.